src/mgr/BaseMgrStandbyModule.cc

```diff
@@ -69,7 +69,10 @@ ceph_config_get(BaseMgrStandbyModule *self, PyObject *args)
     return nullptr;
   }
 
+  PyThreadState *tstate = PyEval_SaveThread();
   std::string value;
+  PyEval_RestoreThread(tstate);
```

Comment:
hmm, I guess you mean to drop GIL after calling get_config

Reply:
indeed.